sdk/ai/azure-ai-inference/azure/ai/inference/models/_patch.py

```diff
@@ -338,7 +338,7 @@ def _deserialize_and_add_to_queue(self, element: bytes) -> bool:
 
         # Convert `bytes` to string and split the string by newline, while keeping the new line char.
         # the last may be a partial "line" that does not contain a newline char at the end.
-        line_list: List[str] = re.split(r"(?<=\n)", element.decode("utf-8"))
+        line_list: List[str] = re.split(r"(?<=\n)", element.decode("utf-8", errors="replace"))  # or errors="ignore"
```

Comment:
@952446418 can you please let me know of a test where the current handling is failing for you, and which package version you're using? This may have been fixed in a recent change in a different way, as I am unable to force an error, so I would like to validate that first before merging this change.